Simple Inventory Management System/Program.cs

```diff
@@ -1,10 +1,39 @@
-﻿namespace Simple_Inventory_Management_System
+﻿using System;
+
+namespace Simple_Inventory_Management_System
 {
-    internal class Program
+    class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello, World!");
+            Inventory inventory = new Inventory();
+
+            while (true)
+            {
+                Console.WriteLine("\n1. Add Product");
+
+                Console.ForegroundColor = ConsoleColor.Yellow;
+                Console.Write("Enter your choice: ");
+                Console.ForegroundColor = ConsoleColor.White;
+                string choice = Console.ReadLine();
+
+                switch (choice)
+                {
+                    case "1":
+                        Console.Write("Enter product name: ");
+                        string name = Console.ReadLine();
+                        Console.Write("Enter product price: ");
+                        decimal price = decimal.Parse(Console.ReadLine());
+                        Console.Write("Enter product quantity: ");
+                        int quantity = int.Parse(Console.ReadLine());
+                        inventory.AddProduct(name, price, quantity);
```

Comment:
Consider using `TryParse` instead of Parse for both decimal and int in order to handle any error in parsing the inputs